webapp-django/crashstats/manage/jinja2/manage/supersearch_fields_missing.html

```diff
@@ -47,7 +47,8 @@
                 </td>
             </tr>
         {% endfor %}
+        </table>
```

Comment:
We must have accidentally lost this in the flurry of recent changes. You'd only notice if you open the page. The effect was kinda amusing. The page footer ended up appearing above the table.